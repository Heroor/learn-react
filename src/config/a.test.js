test('a is running', () => {
  console.log('aaa=====>>>>>aaa')
  expect('aaa=====>>>>>aaa').toMatchSnapshot()
})