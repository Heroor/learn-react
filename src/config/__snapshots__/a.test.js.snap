// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a is running 1`] = `"aaa=====>>>>>aaa"`;
